### ISSUE TITLE
Too many arguments passed in line 30 of index.js

### ISSUE THREAD

**Issue by gspeach** [chrispahm/chartjs-plugin-dragdata#27]
Hello,

I found an issue in your index.js file.

If you look at getElement function you can see this line is passing in 6 arguments, but the function only requires 5.

`let newPos = calcPosition(e,chartInstance,datasetIndex,index,x,y)`

**Comment by chrispahm** [comment]
Hey @gspeach,
thanks for the code review! I just merged the required change in https://github.com/chrispahm/chartjs-plugin-dragdata/pull/28. Beware that I also had to rename the package name (`chartjs-plugin-dragdata` instead of `dragData`) to adhere to the official [chartjs-plugin-id conventions](https://www.chartjs.org/docs/latest/developers/plugins.html#plugin-id). The API didn't change, though (still `dragData: true`). Just make sure to keep that in mind when upgrading to `v1.0` and importing this plugin.

Thanks again,
Christoph
